core/src/main/java/com/eternalcode/core/feature/warp/WarpCommand.java

```diff
@@ -18,11 +19,18 @@ public class WarpCommand {
     private final NoticeService noticeService;
     private final WarpManager warpManager;
     private final TeleportTaskService teleportTaskService;
+    private final WarpInventory warpInventory;
 
-    public WarpCommand(NoticeService noticeService, WarpManager warpManager, TeleportTaskService teleportTaskService) {
+    public WarpCommand(NoticeService noticeService, WarpManager warpManager, TeleportTaskService teleportTaskService, WarpInventory warpInventory) {
         this.noticeService = noticeService;
         this.warpManager = warpManager;
         this.teleportTaskService = teleportTaskService;
+        this.warpInventory = warpInventory;
+    }
+
+    @Execute(required = 0)
+    void warp(Player player, User user) {
+        this.warpInventory.openInventory(player, user.getLanguage());
     }
 
     @Execute(required = 1)
```

Comment:
Proponuję określić uprawnienia dla tego i tamtego. Może to być przydatne, gdy osoba prowadząca serwer chce ustalić, czy gracze mają korzystać tylko z gui, czy tylko z komendy do teleportacji, albo z obu z nich.